figo/Connection.php

```diff
@@ -107,7 +109,8 @@ public function query_api($path, array $data = null, $method='POST', $encode='ht
                         "Content-Type"   => $content_type,
                          "Content-Length" => strlen($data));
 
-        $request = new HttpsRequest($this->apiEndpoint, $this->fingerprints, $this->logger);
+        $request = new HttpsRequest($this->apiUrl['host'], $this->fingerprints, $this->logger);
+        $path = $this->apiUrl['path'] . $path;
```

Comment:
Can `$this->apiUrl['path']` be just `/ `? This might lead to errors then....

Reply:
not possible anymore because of `parse_api_endpoint()`